potassium/potassium.py

```diff
@@ -219,14 +227,25 @@ def handle(path):
 
         @flask_app.route('/__status__', methods=["GET"])
         def status():
+            idle_time = 0
+            inference_time = 0
+            gpu_available = not self._gpu_lock.locked()
+
+            if self._last_inference_start_time != None:
```

Comment:
NIT prefer is not None in python

Reply:
Ah, missed this before merging. I'll add that in another PR and merge it so we get it as well 👍